config/validated_config_db_connector.py

```diff
@@ -94,7 +94,7 @@ def apply_patch(self, gcu_patch, table):
         config_format = ConfigFormat[format.upper()]
 
         try:
-            GenericUpdater().apply_patch(patch=gcu_patch, config_format=config_format, verbose=False, dry_run=False, ignore_non_yang_tables=False, ignore_paths=None)
+            GenericUpdater().apply_patch(patch=gcu_patch, config_format=config_format, verbose=False, dry_run=False, ignore_non_yang_tables=False, ignore_paths=None, sort=False)
```

Comment:
So we assume `apply-patch` in all valiadted_config_db is simple and could be applied without sorting.
I think we can add some comment here about it.

Reply:
Added comment